pkg/identity/identity_test.go

```diff
@@ -81,9 +81,6 @@ func TestToken(t *testing.T) {
 	if token1 != token2 || token1Expire != token2Expire {
 		t.Error("Token updated when not expired")
 	}
-	if *tp.config.Client.CredsToken != token2 {
```

Comment:
Do we know the reason we are removing this ?

Reply:
Hi @bhaskaramVaranasi ,
From the given nested structure `*tp.config.Client.CredsToken`, it looks like the `CredsToken` field has been removed from the recent upgraded packages of `github.com/AthenZ/athenz/clients/go/zms`. As a result, it is no longer supported, hence our test cases are failing. Therefore, we removed the `CredsToken` field.

Reply:
Hi @bhaskaramVaranasi  @zetaj ,
`Client.CredsToken` has been removed from `github.com/AthenZ/athenz@v1.11.51` version. The last it was used in 
`github.com/AthenZ/athenz@v1.11.50` version.

`Below is the screenshot of the Client.CredsToken being used for the final time under client.go file`
<img width="730" alt="Screenshot 2024-06-05 at 12 10 09 PM" src="https://github.com/AthenZ/k8s-athenz-syncer/assets/65297307/047f4af3-8937-4087-948e-e7caf3cd0e60">